mesheryctl/internal/cli/root/environments/view.go

```diff
@@ -89,7 +88,7 @@ mesheryctl environment view --orgID [orgId]
 		var selectedEnvironment environments.EnvironmentData
 
 		if environmentResponse.TotalCount == 0 {
-			utils.Log.Info("No environment(s) found for the given ID: ", orgid)
+			utils.Log.Info("No environment(s) found for the given ID: ", orgID)
```

Comment:
```suggestion
			utils.Log.Info("No environment(s) found for the given ID: ", orgId)
```
Based on changes looks D should be in lower case ?